sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/HttpPostControllerActionShouldValidateInput.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Web.Mvc;
+
+namespace Tests.Diagnostics
+{
+    public class FooBarController1 : Controller
+    {
+        [HttpPost]
+        [ValidateInput(false)] // Noncompliant
+        public ActionResult Purchase1(string input)
+        {
+            return Foo(input);
+        }
+
+        [HttpPost]
+        [ValidateInput(true)] // Compliant - not false
+        public ActionResult Purchase2(string input)
+        {
+            return Foo(input);
+        }
+
+        [HttpPost]
+        [ValidateInput()] // Compliant - not false
+        public ActionResult Purchase3(string input)
+        {
+            return Foo(input);
+        }
+
+        [ValidateInput(false)] // Noncompliant - no HttpPost
```

Comment:
I found the comment _Noncompliant - no HttpPost_ confusing initially. It suggests that that the code is invalid because it HttpPost is missing, whereas what I think it means is that the usage of the attribute is invalid, but we'll still report.
A comment like _Noncompliant - reported even though no HttpPost attribute_

Same for the comment in line 39

Reply:
Done.